test/nexus_test.go

```diff
@@ -1282,15 +1284,37 @@ func TestWorkflowTestSuite_MockNexusOperation(t *testing.T) {
 			},
 			nil,
 		)
-		env.RegisterNexusAsyncOperationCompletion(
+		require.NoError(t, env.RegisterNexusAsyncOperationCompletion(
 			service.Name,
 			dummyOp.Name(),
 			"operation-id",
 			"fake result",
 			nil,
 			0,
-		)
+		))
+
+		env.ExecuteWorkflow(wf, "Temporal")
+		require.True(t, env.IsWorkflowCompleted())
+		require.NoError(t, env.GetWorkflowError())
+		var res string
+		require.NoError(t, env.GetWorkflowResult(&res))
+		require.Equal(t, "fake result", res)
+	})
 
+	t.Run("mock operation reference", func(t *testing.T) {
+		suite := testsuite.WorkflowTestSuite{}
+		env := suite.NewTestWorkflowEnvironment()
+		env.OnNexusOperation(
```

Comment:
Do we have a test that mixes `env.RegisterNexusService(service)` with `env.OnNexusOperation`  registering an operation reference?

Reply:
Added another test mixing it.